task/gitleaks/0.1/gitleaks/gitleaks.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gitleaks
```

Comment:
can you please keep this file in following path `task/gitleaks/0.1/gitleaks.yaml` ?